src/vos/vos_container.c

```diff
@@ -459,8 +466,12 @@ vos_cont_open(daos_handle_t poh, uuid_t co_uuid, daos_handle_t *coh)
 		DP_UUID(cont->vc_id));
 
 exit:
-	if (rc != 0 && cont)
-		cont_decref(cont);
+	if (rc != 0 && cont) {
```

Comment:
line 458 s/Inert/Insert

The wording in the PR is a little confusing. It already wasn't inserting it into the hash.  Isn't the issue that we were calling cont_decref on something that wasn't in the hash table?

Reply:
yes, calling cont_decref on unhashed item will causing the assertion failure shown on the ticket.